src/System.Runtime.CompilerServices.VisualC/src/System.Runtime.CompilerServices.VisualC.csproj

```diff
@@ -4,18 +4,22 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.CompilerServices.VisualC</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
done

---
In reply to: [80331647](https://github.com/dotnet/corefx/pull/11930#discussion_r80331647) [](ancestors = 80331647)